### ISSUE TITLE
[BUG] axolotl atlas annotations has spurious pixels

### ISSUE THREAD

**Issue by alessandrofelder** [brainglobe/brainglobe-atlasapi#344]
**Describe the bug**
Original annotation file contains some spurious pixels

**To Reproduce**
Run the axolotl packaging script and visualise the atlas annotations locally, or run mesh-annotation validation function.

**Expected behaviour**

Axolotl atlas passes validation.

**Log file**
\

**Screenshots**

![image](https://github.com/user-attachments/assets/5b63a1f6-4ddd-47d9-b068-22ed749dacc1)

**Computer used (please complete the following information):**

OS and hardware independent

**Additional context**

Similar to #332

**Comment by alessandrofelder** [comment]
Possibly helpful code snippets:
```python

from skimage.measure import label, regionprops

# Label connected components in the filtered image
labeled_image = label(binary_image)

# Get the properties of each connected component
regions = regionprops(labeled_image)

# Find the largest connected component
largest_region = max(regions, key=lambda region: region.area)

# Create a mask for the largest connected component
largest_mask = labeled_image == largest_region.label

# Apply the mask to the filtered image
binary_image = binary_image * largest_mask

# mask annotation with binary_image
```

**Comment by alessandrofelder** [comment]
Closed by #349
